PURL-TYPES.rst

```diff
@@ -102,6 +102,20 @@ cargo
       pkg:cargo/clap@2.33.0
       pkg:cargo/structopt@0.3.11
 
+chrome
+-----
+``chrome`` for Chrome Browser Extensions:
+
+- The default repository is ``https://chromewebstore.google.com/``.
+- The ``name`` is the extension id - the human readable name often depends on locale.
+- The ``version`` is the package version.
```

Comment:
Is this something that is also tracked with another number internally, like the id above?

Reply:
The places I know if that reference the version number:
- chrome web store page for an extensions lists current version
- the chrome web store `updatecheck` API - returns the current version
- the chrome web store blob URL - files suffixed with a modified form of the version (camel case... plus more?)
- the `manifest.json` has a `version` field

Reply:
NOTE: the extension `id` is NOT the same as the `name` in the `manifest.json` - the name is whatever human-readable name. This allows them to have a primary key that is very collision-free (32 char a-z), and a flexible locale/translation mutable "name" which is presented on web pages / UI.